packages/arm-storage/lib/storageManagementClientContext.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+import * as msRest from "ms-rest-js";
+import * as msRestAzure from "ms-rest-azure-js";
+
+const packageName = "arm-storage";
+const packageVersion = "1.0.0-preview";
+
+export class StorageManagementClientContext extends msRestAzure.AzureServiceClient {
+
+  credentials: msRest.ServiceClientCredentials;
+
+  subscriptionId: string;
+
+  acceptLanguage: string;
+
+  longRunningOperationRetryTimeout: number;
+
+  /**
+   * @class
+   * Initializes a new instance of the StorageManagementClient class.
+   * @constructor
+   *
+   * @param {msRest.ServiceClientCredentials} credentials - Credentials needed for the client to connect to Azure.
+   *
+   * @param {string} subscriptionId - The ID of the target subscription.
+   *
+   * @param {string} [baseUri] - The base URI of the service.
+   *
+   * @param {object} [options] - The parameter options
+   *
+   * @param {Array} [options.filters] - Filters to be added to the request pipeline
+   *
+   * @param {object} [options.requestOptions] - The request options. Detailed info can be found at
+   * {@link https://github.github.io/fetch/#Request Options doc}
+   *
+   * @param {boolean} [options.noRetryPolicy] - If set to true, turn off default retry policy
+   *
+   * @param {string} [options.acceptLanguage] - The preferred language for the response.
+   *
+   * @param {number} [options.longRunningOperationRetryTimeout] - The retry timeout in seconds for Long Running Operations. Default value is 30.
+   *
+   * @param {boolean} [options.generateClientRequestId] - Whether a unique x-ms-client-request-id should be generated. When set to true a unique x-ms-client-request-id value is generated and included in each request. Default is true.
```

Comment:
Do we need to specify types for TypeScript files as well?

Reply:
That's a good point. I don't think we need to specify these properties now that the `options` variable has a type associated with it.